restapi/src/main/java/io/stargate/web/resources/ColumnResource.java

```diff
@@ -146,6 +147,15 @@ public Response addColumn(
     return RequestHandler.handle(
         () -> {
           DataStore localDB = db.getDataStoreForToken(token);
+          Keyspace keyspace = localDB.schema().keyspace(keyspaceName);
+          if (keyspace == null) {
```

Comment:
Why check the keyspace here but not anywhere else? Also why just the keyspace and not the table as well? Currently this is just going to let the query fail on a missing keyspace and return a 400 from that.

Reply:
I'm need the keyspace here because it's now needed below in the `Column.Type.fromCqlDefinitionOf` call, which parse the column type. And the reason it's needed is that CQL types may contain UDT names, so the parser needs the keyspace to find defined UDT. This wasn't done before, but that felt like an oversight.

But since I use that keyspace schema instance, I need an explicit null check or the code below would NPE _before_ we get to the point where the query fails due to the keyspace not existing. And a NPE will not provide the user with a meaningful error message.

So, why not anywhere else? Because I didn't need keyspace schema instance in other places and I was trying to minimize my changes to the rest API code (which I don't know too well tbh).

And why not the table as well? Because I didn't need the Table schema instance, and the query builder will already throw an exception with a meaningful error message if the table is unknown (pre-existing behavior this patch does not modify).